db/seeds.rb

```diff
@@ -86,37 +86,36 @@
 )
 
 # items
-timmys_vaccines_nigeria = people_category.
-                    items.create!(
+timmys_vaccines_nigeria = Item.create!(
                       title: "Timmy's vaccine shots",
                       price: 5000,
                       description: "These are malaria shots for little Timmy.",
-                      retired: false
+                      retired: false,
+                      categories: [people_category]
```

Comment:
This makes a ton of sense. It makes sense why people_category.items.create! doesn't work. Good correction. This allows us to call item.categories now